src/_guides/language/concurrency/index.md

```diff
@@ -462,6 +458,42 @@ is slower when isolates are in different groups.
   Flutter doesn't support `Isolate.spawnUri()`.
 {{site.alert.end}}
 
+## Concurrency on the web
+
+All Dart apps can use `async-await`, `Future`, and `Stream`.
+Isolates, however, are not available on the [Dart web platform][].
+Dart web apps can use [web workers][] to
+run scripts in background threads
+similar to isolates.
+Web workers' functionality and capabilities
+differ somewhat from isolates, though.
+
+For instance, when web workers send data between threads,
+they copy the data back and forth.
+Data copying can be very slow, though,
+especially for large messages. 
+So, rather than copying, isolates _transfer_ the memory
+that holds the message in the exiting isolate
+to the receiving isolate
+(specifically by `Isolate.exit()`).
```

Comment:
Explaining isolate features doesn't feel like it fits here, under a heading of "Concurrency on the web". If it was compared to a similar web feature, then it might be relevant, but it

Drop everything that's just about isolates here, only focus on the web workers, or compare specific parts of web-workers to the similar isolate feature, but only if it is relevant in describing the web worker.

Reply:
**suggestion:** This is a run-on sentence. Try to fix that and eliminate the parenthetical. The content of that parenthetical should stay, just not in parentheses.

Reply:
@lrhn I tried to [rewrite this](https://github.com/dart-lang/site-www/pull/4382/commits/d838272bf305a6ddc917d01ad8401de6d144ec0d#diff-07f3db7a96e07366164a1d378ebb716f2e0be3872e3060612bc74410a56ddd85R476-R487) to not go into so much detail on isolates, but still explain the comparison with web workers.
I skip mentioning `exit` and `run` outright and just said "APIs". Let me know if that doesn't make sense or is worse than before.